### ISSUE TITLE
Fix mod for in-game access

### ISSUE THREAD

**Issue by MichaelMakesGames** [eliasdoehne/stellaris-dashboard#148]
The in-game browser has been removed, breaking the mod.
We could instead add buttons somewhere (bottom right?) that open up the dashboard in the a browser in the Steam Overlay. (If Steam overlay is not available, it opens in the user's default browser.)

See, for example, this button that opens the forums in the Steam Overlay browser:
```
			buttonType = {
				name = "forum"
				quadTextureSprite = "GFX_front_menu_forum"
				position = { x = 390 y = @links_y }
				pdx_tooltip = "FORUM"
				web_link="https://forum.paradoxplaza.com/forum/index.php?forums/stellaris.900/"
			}
```

**Comment by MichaelMakesGames** [comment]
I'm thinking of just replacing the wiki button with a dashboard button
![Screenshot from 2024-05-11 00-00-21](https://github.com/eliasdoehne/stellaris-dashboard/assets/5983927/807a1d46-e79a-4d37-b95d-deb4581bf416)

and adding a link to the wiki in the dashboard itself
![Screenshot from 2024-05-11 00-04-20](https://github.com/eliasdoehne/stellaris-dashboard/assets/5983927/67c9d0c3-3ee5-414c-8053-73cf2c2e49f1)

**Comment by eliasdoehne** [comment]
Looks great! I made the buttons with the graphics borrowed from the game in the earliest version, I was actually surprised that it worked for so long. :smile: 

Replacing the wiki button seems ok to me, but it might be better to add a dedicated button: If someone wants to quickly access the wiki to check an event outcome or similar, it could be annoying to add the extra click and it also won't work if they run the game without the dashboard. But I also don't know how many people really use that, I don't think I ever did

**Comment by MichaelMakesGames** [comment]
I also haven't ever used that button either

Replacing the wiki button will make it easier to keep the mod updated, and to make compatibility patches for other UI mods (there have been requests for compat with UI Overhaul Dynamic). Plus I really dislike Stellaris GUI modding, so the more I can minimize it the better, haha

I'd be inclined to just replace the wiki button as a start, and see if anyone complains about it

**Comment by MichaelMakesGames** [comment]
Also a note: once the browser is open, you can bring it up with shift+tab, and it retains whatever tabs we're open. So it's only an extra click once if you keep the tab open. I'll add a note about this to the mod description.

(Annoyingly, if you keep the tabs open, close the Steam overlay, then click the button again, it opens a duplicate tab, instead of focusing the matching tab. That behavior is out of our control

**Comment by MichaelMakesGames** [comment]
Fixed by #150
